### PR TITLE
feat(route): 中证网中证快讯

### LINKED ISSUE #14420: 【中证网】 路由抓取返回乱码，瘉鍒稿
### 路由地址

```routes
/cs/news/:caty
```

### 完整路由地址

```fullroutes
https://rsshub.app/cs/news/zzkx
```

### 相关文档

https://docs.rsshub.app/zh/routes/finance#zhong-zheng-wang

### 预期是什么？

返回正常文字，utf-8

### 实际发生了什么？

结果标题和内容都变成乱码

娴烽€氳瘉鍒稿姪鍔涘帵闂ㄥ競鏀块泦鍥㈠強鍘

### 部署

RSSHub 演示 (https://rsshub.app)

### 部署相关信息

_No response_

### 额外信息

```shell
感谢
```

### 这不是重复的 issue

- [X] 我已经搜索了 [现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

### PR DESCRIPTION
**Pull request by nczitzk**
## Involved Issue / 该 PR 相关 Issue

Close #14420

## Example for the Proposed Route(s) / 路由地址示例

```routes
/cs/zzkx
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
- [x] Full text / 全文获取
  - [x] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/cs/zzkx.js [hunk 1]
```diff
@@ -1,45 +1,34 @@
...
 };
 
 module.exports = async (ctx) => {
-    const cfg = config[ctx.params.caty];
-    if (!cfg) {
-        throw new Error('Bad category. See <a href="https://docs.rsshub.app/routes/finance#zhong-zheng-wang-zi-xun">docs</a>');
-    }
-
-    const currentUrl = url.resolve(rootUrl, cfg.link);
+    const currentUrl = url.resolve(rootUrl, config.link);
```
Comment (TonyRL) [inline lib/v2/cs/zzkx.js:16]:
```suggestion
    const currentUrl = new URL(config.link, rootUrl).href;
```

File: lib/v2/cs/zzkx.js [hunk 2]
```diff
@@ -1,45 +1,34 @@
...
+
+    const $ = cheerio.load(response.data);
+    const list = $(config.query)
         .slice(0, 20)
         .map((_, item) => {
             item = $(item);
             const a = item.find('a');
             return {
                 title: item.find('h3').text() || a.text(),
                 link: url.resolve(currentUrl, a.attr('href')),
```
Comment (TonyRL) [inline lib/v2/cs/zzkx.js:30]: https://github.com/DIYgod/RSSHub/pull/14424/files#r1481797408

Comment (TonyRL) [inline lib/v2/cs/zzkx.js:30]:
```suggestion
                link: new URL(a.attr('href'), currentUrl).href,
```

File: lib/v2/cs/zzkx.js (hunk 1 above)
Comment (nczitzk) [reply lib/v2/cs/zzkx.js:16]: 👌

**Pull request merged**